rdflib/plugins/parsers/notation3.py

```diff
@@ -1570,8 +1585,8 @@ def strconst(self, argstr, i, delim):
             if ch == delim1:
                 j = i
                 continue
-            elif ch in ('"', "'") and ch != delim1:
-                ustr = ustr + ch
+            elif ch in "\"'" and ch != delim1:
```

Comment:
To me the original looks more readable and - at least in cpython - it is actually faster. Didn't test cython though.

Reply:
Indeed the effect of replacing "x = x + y" by "x += y" is very small. However, the other changes have a significant effect on my platform.